test/release/examples/primers/Makefile

```diff
@@ -30,6 +30,16 @@ TARGETS = \
 	varargs \
 	variables \
 
+#
+# Only make FFTW if FFTW_DIR is set or we can find the fftw library
+#
+ifdef FFTW_DIR
+	TARGETS += FFTWlib
+	FFTW_OPTS = -I$(FFTW_DIR)/include -L$(FFTW_DIR)/lib
+else ifneq ($(shell /sbin/ldconfig -p | grep -i fftw),)
```

Comment:
There should probably be a check that /sbin/ldconfig is available. I think it will be available on most linux systems, but unlikely to exist on mac (and maybe cygwin). For example, on my mac it prints this error message right now:

     /bin/sh: /sbin/ldconfig: No such file or directory

This change seems to fix that:

https://github.com/chapel-lang/chapel/commit/3951c501d55a31a4b75aae81d677fc64d9430b0f